joomla/joomla-cms
# [6.2] Rebuild the image filter to a registry allowing custom filters outside the main namespace

Pull Request for Issue #31804 .

### Summary of Changes
Refactor the image filter class to have custom namespaces by using a registry for loading classes.

#### Why is this b/c changing at this stage?
Because there is a poor user experience for third party extensions who'd have to custom autoload code to add custom filters (as evidenced by the fact we were doing this in our own tests until yesterday) 

### Testing Instructions
- Check loading a custom image class with a custom namespace
- Check drone tests continue pass  when loading a custom registry

### Documentation Changes Required
Yup - refactor the namespaced stuff.
